### ISSUE TITLE
[Bug]: mo fails to start in docker mode

### ISSUE THREAD

**Issue by aronchanisme** [matrixorigin/matrixone#11206]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): nightly-6a5db18a
- Hardware parameters: n.a.
- OS type: n.a.
- Others: n.a.
```

### Actual Behavior

Start mo-service in docker mo will fail in recent commit ids,
<img width="1258" alt="企业微信截图_ee07e858-59f2-4027-b40c-dc1e1a09bd61" src="https://github.com/matrixorigin/matrixone/assets/46433151/177608de-70a0-40da-95f0-369067b76c4f">

From docker logs of the container, it is clear that `etc/launch-tae-CN-tae-DN/` is missing due to previous clean up of configuration folders,
<img width="993" alt="企业微信截图_134a0ede-71a6-45e2-a434-81b4637e30b3" src="https://github.com/matrixorigin/matrixone/assets/46433151/ad0193d3-2c24-481e-9e5a-f81b7e6ac6b3">

Please kindly help correct the startup command for docker container mode.

### Expected Behavior

mo starts normally

### Steps to Reproduce

```Markdown
1. Get a latest mo docker image
```bash
docker pull registry.cn-shanghai.aliyuncs.com/matrixorigin/matrixone:nightly-commitnumber
```

2. Start a container from this image
```bash
docker run -d -p 6001:6001 --name matrixone --privileged=true registry.cn-shanghai.aliyuncs.com/matrixorigin/matrixone:nightly-commitnumber
```

### Additional information

https://docs.matrixorigin.cn/0.8.0/MatrixOne/Get-Started/install-on-macos/install-on-macos-method3/#1-docker

**Comment by sukki37** [comment]
fixed by #11184 , @aronchanisme you can try with matrixorigin/matrixone:nightly-b4fbfeed

**Comment by aronchanisme** [comment]
Fix's confirmed.
```bash

root@test0:~# docker exec -it f1f1239e41e1 bash
root@f1f1239e41e1:/# ps -ef 
UID          PID    PPID  C STIME TTY          TIME CMD
root           1       0 26 06:56 ?        00:00:46 /mo-service -debug-http=:12345 -launch /etc/quickstart/launch.toml
root          34       0  1 06:59 pts/0    00:00:00 bash
root          42      34  0 06:59 pts/0    00:00:00 ps -ef
root@f1f1239e41e1:/# cat /etc/quickstart/launch.toml
logservices = [
    "./etc/launch/log.toml",
]

dnservices = [
    "./etc/launch/dn.toml"
]

cnservices = [
    "./etc/launch/cn.toml"
]
root@f1f1239e41e1:/# exit
exit
root@test0:~# docker ps 
CONTAINER ID   IMAGE                                     COMMAND                  CREATED         STATUS         PORTS                     NAMES
f1f1239e41e1   matrixorigin/matrixone:nightly-b4fbfeed   "/mo-service -debug-…"   3 minutes ago   Up 3 minutes   0.0.0.0:36001->6001/tcp   matrixone
root@test0:~# docker images 
REPOSITORY               TAG                IMAGE ID       CREATED       SIZE
matrixorigin/matrixone   nightly-b4fbfeed   1e4dc3f49974   4 hours ago   227MB

root@test0:~# mysql -u dump -p111 -h127.0.0.1 -P36001
Welcome to the MariaDB monitor.  Commands end with ; or \g.
Your MySQL connection id is 8
Server version: 8.0.30-MatrixOne-v0.8.0 MatrixOne

Copyright (c) 2000, 2018, Oracle, MariaDB Corporation Ab and others.

Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.

MySQL [(none)]> show databases;
+--------------------+
| Database           |
+--------------------+
| mo_task            |
| information_schema |
| mysql              |
| system_metrics     |
| system             |
| mo_catalog         |
+--------------------+
6 rows in set (0.004 sec)

MySQL [(none)]> 

```

Steps to reproduce
```bash
# Step 1. 
docker pull matrixorigin/matrixone:nightly-b4fbfeed

# Step 2.
docker run -d -p 6001:6001 --name matrixone --privileged=true  matrixorigin/matrixone:nightly-b4fbfeed

# Step 3.
mysql -u dump -p111 -h127.0.0.1 -P36001
```
